core/src/main/java/org/axonframework/queryhandling/QueryUpdateEmitter.java

```diff
@@ -53,6 +63,11 @@
 
     /**
      * Emits given incremental update to subscription queries matching given query type and filter.
+     * <p>
+     * Note that if this operation is called whilst a current {@link org.axonframework.messaging.unitofwork.UnitOfWork}
+     * is active and {@link org.axonframework.messaging.unitofwork.UnitOfWork.Phase#STARTED}, then this operation should
+     * be performed on the {@link org.axonframework.messaging.unitofwork.UnitOfWork.Phase#AFTER_COMMIT} phase. Doing so
+     * ensures this operation is done after a calling task has rounded up its process,
```

Comment:
`.` instead of `,` at the end. 

A general remark about this JavaDoc: It looks to technical to me, developers may be confused when it's going to be invoked. I would increase a level of abstraction and mention _calling within a message handling method_ because IMO it's a bit more understandable than having developers figuring out what it means for a unit of work to be active and in STARTED phase.

Please consider this comment as a suggestion :)

Reply:
Taken note of this. Changed the javadoc somewhat to (hopefully) be a bit less technical.